CHANGES.md

```diff
@@ -3,9 +3,15 @@
 
 # v 1.32
 
+* Breaking changes
+  * Changed dAbstractSynchronizer.createSingleLayerCounterparts Plugin API
+    in existing plugins the function has to be modified.
```

Comment:
It now takes an object `{layer: !ol.layer.Base, parents: Array<ol.layer.Group>}`. The parents are ordered with the first parent coming first and the last parent coming last.